### PR TITLE
chore(core-dumps): add core dump capabilities

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description

 - Enable debug symbols
 - Add package managers to main image
 - Add gdb and tar to container
 - Add script to fetch core dump

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: fetch-core-dump.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# get sensor pod name
+sensor_pod_name=$(kubectl -n stackrox get pods -l app=sensor -o custom-columns=NAME:.metadata.name | tail -n 1)
```
Comment (vikin91) [inline fetch-core-dump.sh:4]:
```suggestion
sensor_pod_name="$(kubectl -n stackrox get pods -l app=sensor -o=jsonpath='{.items[0].metadata.name}')"
```

File: fetch-core-dump.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+# Create core dump
+kubectl exec ${sensor_pod_name} -- /bin/bash -c "cd /var/cache/stackrox && /bin/bash -c 'gcore 1'"
+
+# Copy core-dump
+kubectl cp ${sensor_pod_name}:/var/cache/stackrox/core.1 ./core.1
```
Comment (vikin91) [inline fetch-core-dump.sh:10]: Let's maybe add some names to it and add the quotes
```suggestion
kubectl cp "${sensor_pod_name}:/var/cache/stackrox/core.1" "core-dump.1.${sensor_pod_name}"
```

**Review by vikin91 — APPROVED** [review]
That looks good, but I wonder how this should be used? I am guessing that the customer should simply run the script against the cluster and a new file would appear, right?

Have you verified that the produced file is useful for us? What can we do with it?

**Review by lvalerom — APPROVED** [review]

File: fetch-core-dump.sh (hunk 1 above)
Comment (SimonBaeumer) [reply fetch-core-dump.sh:4]: done

File: fetch-core-dump.sh (hunk 2 above)
Comment (SimonBaeumer) [reply fetch-core-dump.sh]: done

**Pull request merged**